website/src/models/group.js

```diff
@@ -164,7 +175,7 @@ GroupSchema.methods.sendChat = function(message, user){
   // Kick off chat notifications in the background.
   var lastSeenUpdate = {$set:{}, $inc:{_v:1}};
   lastSeenUpdate['$set']['newMessages.'+group._id] = {name:group.name,value:true};
-  if (NO_CHAT_NOTIFICATIONS.indexOf(group._id) !== -1 || group.memberCount > 5000) {
+  if (NO_CHAT_NOTIFICATIONS.indexOf(group._id) !== -1 || group.isTooLargeForChatNotifications) {
```

Comment:
@cridenour The NO_CHAT_NOTIFICATIONS was moved to a const further up the file.

@khipkin Sorry, we still haven't reviewed this. :( Thanks for keeping it up to date with develop. I'll try to get to this this weekend if no one else does it sooner.